server/applications.json

```diff
@@ -1358,6 +1358,64 @@
                 }
             ]
         },
+        "mochapro": {
+            "enabled": true,
+            "host_name": "mochapro",
+            "environment": "{}",
+            "variants": [
+                {
+                    "name": "12-0-0",
+                    "label": "2025",
```

Comment:
Why label is `2025` and name is `12-0-0`? 🙂

Reply:
they have the version (which is 12.0.0) but apart of api and some docs, they mostly use the year version. I thought that this way it will be more defined.